tools/cdk-build-tools/lib/package-info.ts

```diff
@@ -126,6 +126,13 @@ export interface CDKBuildOptions {
    */
   pre?: string[];
 
+  /**
+   * An optional command (formatted as a list of strings) to run after building
+   *
+   * (Typically a code generator)
```

Comment:
I would expect code generation to usually happen in "pre", but that's just me.

Reply:
I suppose thats true. I'll rephrase.